packages/palette/tsconfig.ios.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "./*": ["./*.ios", "./*"],
```

Comment:
:ceiling-cat:

Can I suggest adding a link to https://github.com/microsoft/TypeScript/issues/8328#issuecomment-219583889 in a comment here? This is a fairly advanced bit of config and it might save someone some googling one day.

:ceiling-cat-out: